ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/admin/console/components/build/clientlibs/js/build.js

```diff
@@ -258,6 +258,8 @@ $(function () {
                         });
                         $buildLog.append('<h4>Approximate size of the assets in the package: ' + bytesToSize(data.dataSize) + '</h4>');
                         scrollLog();
+                    } else {
+                        $buildLog.append(<h4>There are no assets in the package</h4>);
                     }
```

Comment:
![image](https://user-images.githubusercontent.com/70597105/136100519-dcaf05b0-d2fd-4652-ae16-a828849662f6.png)
I think you need something like this

Reply:
Uh ohk.